polyfill/lib/calendar.mjs

```diff
@@ -83,7 +85,7 @@ export class Calendar {
   fields(fields) {
     if (!ES.IsTemporalCalendar(this)) throw new TypeError('invalid receiver');
     const fieldsArray = [];
-    const allowed = new Set([
+    const allowed = new OriginalSet([
       'year',
       'month',
       'monthCode',
```

Comment:
I think this is where the only rebase conflict is; you'll have to remove the time units from this list.